mlflow/shap.py

```diff
@@ -459,6 +459,11 @@ def save_explainer(
     with open(os.path.join(path, conda_env_subpath), "w") as f:
         yaml.safe_dump(conda_env, stream=f, default_flow_style=False)
 
+    pip_req_subpath = "requirements.txt"
+    pip_deps = _get_additional_pip_dep(conda_env)
+    with open(os.path.join(path, pip_req_subpath), "w") as f:
```

Comment:
Do we need requirements.txt for SHAP? cc: @harupy ?

Reply:
I think we do :)